spec/controllers/admin/users_controller_spec.rb

```diff
@@ -150,15 +150,6 @@ def self.name
 
         expect(response.body).not_to have_selector('body.active_admin')
       end
-
-      it 'is forbidden for account publisher' do
-        account = create(:account)
-
-        sign_in(create(:user, :publisher, on: account))
-        get(:quota_state, params: {account_id: account})
-
-        expect(response.body).to have_selector('.user_quota_not_allowed')
-      end
```

Comment:
There _is_ a policy spec for this, but most of Pageflow's controller specs still test the border between allowed and forbidden stuff (e.g. to make sure the `authorize` call that invokes the tested policy is in there at all). Also this tests a different code path of the controller action than the other two.